git/homework.py

```diff
@@ -15,23 +15,32 @@ def is_two_object_has_same_value(first: Any, second: Any) -> bool:
     If @first and @second has same value should return True
     In another case should return False
     """
-    pass
+    if first == second:
+        return True
+    else:
+        return False
 
 
 def is_two_objects_has_same_type(first: Any, second: Any) -> bool:
     """
     If @first and @second has same type should return True
     In another case should return False
     """
-    pass
+    if type(first) == type(second):
+        return True
+    else:
+        return False
 
 
 def is_two_objects_is_the_same_objects(first: Any, second: Any) -> bool:
     """
     If @first and @second has same type should return True
     In another case should return False
     """
-    pass
+    if id(first) == id(second):
+        return True
+    else:
+        return False
 
```

Comment:
The same that in #1.

Reply:
use operator `is` instead

Reply:
used is instead of id() but it's doubtful

Reply:
Read about `is`)